air_quality_monitoring/raqmn/raqmn_api/views.py

```diff
@@ -90,15 +84,17 @@ def get_all_sensors():
 	r = requests.post(query_url, query)
 	return r.content
 
-def get_data_by_uuid_helper(uuid):
+def get_data_by_uuid_helper(uuid, hours):
 	"""
 	Returns the data of sensor specified by uuid of last 30 days
-	TODO : Return the data given time and uuid both, right now only uuid is considered and by default last 30 days data is sent
+	TODO : Return the data given time in days,hours,minutes
 	"""
 	parser = HTMLParser()
 	uuid = parser.unescape(uuid)
 	query_url = 'http://localhost:8079/api/query'
-	query = "select data in (now -30d, now) where uuid='"+uuid+"'"
+	query = "select data in (now -"+hours+"h, now) where uuid='"+uuid+"'"
```

Comment:
So this is the main part of the change, right? Cool that you can change it just by making a small edit to a query.